src-js/instantsearch.js

```diff
@@ -0,0 +1,85 @@
+import * as debounce from 'lodash.debounce';
+
+// Instant search initialization
+export function init () {
+  var search = instantsearch({
+    appId: 'U0RXNGRK45',
+    apiKey: 'dd49f5b81d238d474b49645a4daed322', // search-only API Key; safe for front-end code
+    indexName: 'documentation',
+    routing: true
+  });
+
+  // adding conditions to filter search
+  search.addWidget(
+    instantsearch.widgets.configure({
+      filters: "collection: cci2"
+    })
+  );
+
+  // initialize SearchBox
+  search.addWidget(
+    instantsearch.widgets.searchBox({
+      container: '#search-box',
+      cssClasses: {
+        input: 'instantsearch-search'
+      },
+      placeholder: 'Search Documentation',
+      queryHook: debounce(function (query, searchFunction) { searchFunction(query); setTimeout(renderResults, 100); }, 500, { 'leading': true, 'trailing': true, 'maxWait': 1000 }) // method to throttle search requests
```

Comment:
There's probably a concern here that I'm not thinking of 🙈But why do we use `setTimeout` here?